server/org.wso2.developerstudio.eclipse.carbonserver44/src/org/wso2/developerstudio/eclipse/carbonserver44/internal/CarbonServerBehavior44.java

```diff
@@ -248,7 +248,10 @@ protected void addServletTransportPorts(List<Integer> ports, String carbonXmlPat
 			int offSet = Integer.parseInt((String)xPath.evaluate("/Server/Ports/Offset", document, XPathConstants.STRING));
```

Comment:
can't we keep a constant for "/Server/Ports/Offset" ,as i feel this will be re-using.